static/js/markdown_config.js

```diff
@@ -29,8 +29,10 @@ export const get_helpers = () => ({
     // user stuff
     get_actual_name_from_user_id: people.get_actual_name_from_user_id,
     get_user_id_from_name: people.get_user_id_from_name,
+    get_by_email: people.get_by_email,
```

Comment:
`get_by_email` is an ambiguous name without the `people` qualifier.

Reply:
I changed the helper name to `get_user_by_email`.